### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by stingion**
I've provided some minor bug fixes

--------------------------------------
P. S. Thank you for good microservice-based education project.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -162,7 +162,7 @@ services:
       config:
         condition: service_healthy
     ports:
-      - 9000:8080
+      - 9000:9000
```
Comment (sqshq) [inline docker-compose.yml]: why do you think this port should be changed? as far as I can see monitoring service exposes 8080

Comment (stingion) [reply docker-compose.yml]: In docker container monitoring application is starting on 9000 port. Run up it all to be sure.

Comment (sqshq) [reply docker-compose.yml]: It starts on 8080. No code changes needed.
If you used public docker images from docker hub, please pull the latest ones.

Comment (stingion) [reply docker-compose.yml]: Yes, now It works. Also consider to change "jdk: oraclejdk8" to "jdk: openjdk8" in .travis to fix travis build process.

If you have a time can you, please, add ELK logging (based on https://github.com/sqshq/ELK-docker) in project or in separate branch. It's interesting to see best practice for microservices.

**Pull request closed without merging**